app/uk/gov/hmrc/advancevaluationrulings/models/RejectReason.scala

```diff
@@ -16,21 +16,16 @@
 
 package uk.gov.hmrc.advancevaluationrulings.models
 
-import play.api.libs.json.{Format, Json}
-import uk.gov.hmrc.advancevaluationrulings.models
+import enumeratum.{Enum, EnumEntry, PlayJsonEnum}
 
-object RejectReason extends Enumeration {
-  type RejectReason = Value
-  val APPLICATION_WITHDRAWN, ATAR_RULING_ALREADY_EXISTS, DUPLICATE_APPLICATION, NO_INFO_FROM_TRADER, OTHER = Value
+sealed abstract class RejectReason(override val entryName: String) extends EnumEntry
```

Comment:
I don't know if `entryName` semantically describes what this string represents for a `RejectReason`. Any thoughts on what a better name here could be?

Reply:
I'd often just use `value` for these 😆